src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = (x) => {
+    count++;
+
+    const count2 = count;
+
+    return function(y) {
+      if ((count2 > 3 && count2 % 2 === 0)) {
```

Comment:
```suggestion

    return function(y) {
      if ((count > 3 && count % 2 === 0)) {
```
Для чого ця дублююча змінна count2? Ти можеш використовувати прямо count